docs/core/development/policy/coding-standard.rst

```diff
@@ -617,12 +617,11 @@ An attribute (or function, method or class) should be considered private when on
 Python 3
 --------
 
-Twisted is being ported to Python 3, targeting Python 3.5+.
+Twisted is being ported to Python 3, targeting Python 3.6+.
 Please see :doc:`Porting to Python 3 </core/howto/python3>` for details.
 
-All new modules must be Python 2.7 & 3.5+ compatible, and all new code to ported modules must be Python 2.7 & 3.5+ compatible.
-New code in non-ported modules must be written in a 2.7 & 3.5+ compatible way (explicit bytes/unicode strings, new exception raising format, etc) as to prevent extra work when that module is eventually ported.
-Code targeting Python 3 specific features must gracefully fall-back on Python 2 as much as is reasonably possible (for example, Python 2 support for 'async/await' is not reasonably possible and would not be required, but code that uses a Python 3-specific module such as ipaddress should be able to use a backport to 2.7 if available).
+All new modules must be Python 3.6+ compatible, and all new code to ported modules must be 3.6+ compatible.
```

Comment:
Not sure if this paragraph has any current value.
It was created while we were still supporting 2.7.

I think that now, is common sense that any code should work on the minimum supported Python 3 version.

So, either remove this paragraph or replace it with something like:

```suggestion
All new modules must be compatible with the minimum supported Python version.

```

Reply:
yeah I think this being removed makes the most sense, CI already enforced the relevant python versions

```suggestion
```